common/src/Microsoft.Azure.IIoT.Hub.Module.Framework/src/Clients/EdgeletClient.cs

```diff
@@ -29,10 +30,10 @@ public sealed class EdgeletClient : ISecureElement {
         /// <param name="logger"></param>
         public EdgeletClient(IHttpClient client, IJsonSerializer serializer,
             ILogger logger) : this(client, serializer,
-            Environment.GetEnvironmentVariable("IOTEDGE_WORKLOADURI")?.TrimEnd('/'),
-            Environment.GetEnvironmentVariable("IOTEDGE_MODULEGENERATIONID"),
-            Environment.GetEnvironmentVariable("IOTEDGE_MODULEID"),
-            Environment.GetEnvironmentVariable("IOTEDGE_APIVERSION"),
+            Environment.GetEnvironmentVariable(IoTEdgeVariables.IOTEDGE_WORKLOADURI)?.TrimEnd('/'),
```

Comment:
This trim end should be in the class instead of here.

Reply:
Fixed.